docs/pipeline/actions/probabilisticsampler.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Probabilistic Sampler"
+sidebarTitle: "Probabilistic Sampler"
+---
+
+The "Probabilistic Sampler" Odigos Action supports probabilistic sampling based on a configured sampling percentage applied to the TraceID.
+
+### Use Cases
+
+#### Cost Reduction
+
+- Some vendors charge based on the amount of data ingested. For self hosted destinations, the cost is correlated to the use of cloud resources which grows with the based on the amount of data you process and store. By Reducing traces volumes, you can reduce the amount of data ingested and reduce costs.
+
+#### Usability
+
+- Performance Trend Analysis - In certain scenarios, users may seek to collect sufficient data to understand performance trends without overwhelming the system with excessive telemetry data. For example, track latency trend.
+- Telemetry Volume Management - When experiencing a sudden influx of telemetry data, probabilistic sampling can temporarily reduce data volume sent to your data destination.
+
+
+### Mechanism
+
+This action receives a sampling percentage as a float to configure the sampler.
+
+### Basic Example
+
+The basic example below shows how to add a sampler with 
+
+Create a file named `probabilistic-sampler.yaml` with the following content:
+
+```yaml
+apiVersion: actions.odigos.io/v1alpha1
+kind: ProbabilisticSampler
+metadata:
+  name: example-probabilistic-sampler
+  namespace: odigos-system
+spec:
+  actionName: "configure probabilistic_sampler"
+  sampling_percentage: "15"
+  signals:
+    - TRACES
+```
+
+Apply the action to the cluster:
+
+```bash
+kubectl apply -f probabilistic-sampler.yaml
+```
+
+### Full Action Options
+
+The full list of options available for the "ProbabilisticSampler" action are:
+
+- `sampling_percentage` (required): Percentage at which items are sampled; >= 100 samples all items, 0 rejects all items.
```

Comment:
I don't think we should promote for users to use a value larger than 100 in the docs :)

```suggestion
- `sampling_percentage` (required): Percentage at which items are sampled; 100 samples all items, 0 rejects all items.
```